test/service/test_manifest.py

```diff
@@ -1037,8 +1039,9 @@ def test_compact_metadata_cache(self, _time_until_object_expires: MagicMock):
                         else:
                             _time_until_object_expires.assert_called_once()
                         _time_until_object_expires.reset_mock()
-                        header = response.headers['Content-Disposition']
-                        value, params = cgi.parse_header(header)
+                        header_type = 'Content-Disposition'
+                        header = response.headers[header_type]
+                        value, params = parse_header(header_type, header)
```

Comment:
```suggestion
                        name = 'Content-Disposition'
                        value = response.headers[name]
                        value, params = parse_header(name, value)
```

As discussed in PL. This is the *name* of the header, not its type. *Type* would be a more general category such as *request header* or *response header*.